src/main/scala/com/google/cloud/spark/bigquery/SparkBigQueryOptions.scala

```diff
@@ -23,10 +28,15 @@ import org.apache.spark.sql.types.StructType
 case class SparkBigQueryOptions(
     tableId: TableId,
     parentProject: String,
+    credentials: Option[String] = None,
```

Comment:
Why not specify a file name instead of the string directly here? It at least is more resilient to potential leaks in logs.
Also, please make parameter name more descriptive. Something like "serviceAccountKeyFile".

Reply:
Ideal would be to have both options, but string is sometimes the only option when you don't have any shared file system that is "secure" to upload the file.

Reply:
In this case, can you please add both options (exclusive)?